app/js/components/manageApps/ManageApps.jsx

```diff
@@ -85,11 +85,16 @@ export default class ManageApps extends React.Component {
   }
 
   handleApplist() {
+    const resultData = [];
     this.apiHelper.get('/owa/applist').then(response => {
+      resultData.push({
+        appDetails: response[0],
+        install: false
```

Comment:
You are declaring 'resultData with 'const' yet you are changing its value, it  looks like you should be use 'let' instead. Check out this similar case https://stackoverflow.com/questions/40825010/const-or-let-in-react-component
There could other similar in your code, please check through.

Reply:
I think it's right to use `const` in this context. `resultData` is not totally been reassigned, it's just an addition of values. And by the way, `eslint` will flag it if you use `let` 🙂

Reply:
Thanks @jacksono @andela-dbamidele Following up with the thread, I only mutated the data in `resultData` and didn't re-assign it. `const` only prevents re-assignment, so it pretty much works for this use case.